README.md

```diff
@@ -1 +1,45 @@
 This is an Eclipse plugin for building and deploying Google Cloud Platform applications.
+
+* Development
+
+** Import into Eclipse
+
+*** Requirements
+
+1. Eclipse Mars or later
+
+1. The [m2e plugin](http://www.eclipse.org/m2e/) installed to import the project into Eclipse
+
+1. JDK 7
+
+*** Steps to import
+
+1. Setup JDK 7 in Eclipse
+  1. Select `Window/Preferences` (on Mac `Eclipse/Preferences`)
+  1. Under `Java/Installed JREs` click `Add`
+  1. Select Standard VM and click `Next`
+  1. Select the folder that contains the JDK 7 installation by clicking
+     `Directory`
+  1. Set the name to `jdk7`
```

Comment:
To be honest, I think it's better to name the VM with the vendor and build number. E.g., OpenJDK 1.7_b39. Really helpful if you hit something that seems to be a JIT or JVM bug.